### ISSUE TITLE
Dependency based word2vec model accuracy

### ISSUE THREAD

**Issue by scottyli** [spacy-io/spaCy#81]
Do you have accuracy tests for the dependency based word2vec model?

In particularly, I think people might be interested in how it scores against these word associations - http://word2vec.googlecode.com/svn/trunk/questions-words.txt

In my own experiments, I've found the pretrained [google news corpus](https://code.google.com/p/word2vec/#Pre-trained_entity_vectors_with_Freebase_naming) to be ~55% accurate, whereas the pretrained glove model [(Common Crawl (42B tokens))](http://nlp.stanford.edu/projects/glove/) was ~75%.

I used gensim's ```model.accuracy``` [method](https://radimrehurek.com/gensim/models/word2vec.html#gensim.models.word2vec.Word2Vec.accuracy)

Would be great to get stats on your model!

**Comment by honnibal** [comment]
Still no accuracy evaluation for these vectors unfortunately, but going to close this in the interest of clearing out old issues.
